geode-core/src/main/java/org/apache/geode/internal/InternalDataSerializer.java

```diff
@@ -2435,26 +2435,28 @@ public static Version getVersionForDataStream(DataInput in) {
   }
 
   /**
-   * Get the {@link Version} of the peer or disk store that created this {@link DataInput}. Returns
+   * Get the {@link Version} of the peer or disk store that created this
+   * {@link DataInput}. Returns
    * null if the version is same as this member's.
    */
   public static Version getVersionForDataStreamOrNull(DataInput in) {
     // check if this is a versioned data input
     if (in instanceof VersionedDataStream) {
-      return ((VersionedDataStream) in).getVersion();
+      return (Version) ((VersionedDataStream) in).getVersion();
```

Comment:
All the casts to `(Version)` in this file are superfluous. Recommend removal.

Reply:
I think those casts must have crept in when I was trying to leave a subclass of Version in geode-core.  I'll remove them.

Reply:
done